api/mocks/mocks.go

```diff
@@ -0,0 +1,7 @@
+package mocks
+
+import "github.com/onflow/flow-go-sdk/access"
+
+type FlowAccessClient interface {
+	access.Client
+}
```

Comment:
```suggestion
type FlowAccessClient access.Client
```

Reply:
Unfortunately, this does not work for mockery. I think it needs an interface definition to work, and not a type "alias".
I had a look at flow-go(https://github.com/onflow/flow-go/commit/02e4142506d6b413a3b87fd5d7b3096f4f44656a), and the Proxy interface is how they mock implementations for interfaces defined in 3rd party packages.

Reply:
Interesting.